src/runtime.rs

```diff
@@ -307,6 +328,20 @@ impl Class {
 
     }
 
+    /// Checks whether this class conforms to the specified protocol.
+    pub fn conforms_to_protocol(&self, proto: &Protocol) -> bool {
```

Comment:
I was just following the original ObjC api when I wrote this, but I agree that `conforms_to` is cleaner. I think that it's the right name for this method.